x-pack/plugins/security_solution/public/common/components/exceptions/viewer/exception_item/index.test.tsx

```diff
@@ -12,17 +12,18 @@ import { mount } from 'enzyme';
 import { ExceptionItem } from './';
 import { getExceptionListItemSchemaMock } from '../../../../../../../lists/common/schemas/response/exception_list_item_schema.mock';
 import { getCommentsArrayMock } from '../../../../../../../lists/common/schemas/types/comment.mock';
+import { getMockTheme } from '../../../../lib/kibana/kibana_react.mock';
 
 jest.mock('../../../../lib/kibana');
 
-const mockTheme = {
+const mockTheme = getMockTheme({
   eui: {
     euiColorDanger: '#ece',
     euiColorLightestShade: '#ece',
     euiColorPrimary: '#ece',
-    euiFontWeightSemiBold: 'bold',
+    euiFontWeightSemiBold: 1,
```

Comment:
why the change here?

Reply:
euiFontWeightSemiBold is a number! It's probably 400 or so in the light / dark theme